support-frontend/assets/pages/digital-subscription-landing/components/cta.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+
+import PaymentSelection from 'pages/digital-subscription-landing/components/paymentSelection/paymentSelection';
+
+import './digitalSubscriptionLanding.scss';
+
+
+type PropTypes = {
+  dailyEditionsVariant: boolean,
+}
+
+const CallToAction = (props: PropTypes) => (
+  <div className="call-to-action__container">
+    <div className="hope-is-power--centered">
+      <h2>Choose one of our special offers and subscribe today</h2>
+      <h3>After your 14-day free trial, your subscription will begin automatically and you can cancel any time</h3>
+      <PaymentSelection dailyEditionsVariant={props.dailyEditionsVariant} pageType="A" />
```

Comment:
The `PaymentSelection` component doesn't use the `dailyEditionsVariant` prop. You should probably change it to use it  and remove `pageType` (this would mean we only have the one way of determining which variant to show) or alternatively remove `dailyEditionsVariant` from the props passed in here (I think this is a less good option).